### PR TITLE
Shontzu/87863/deriv navigation icon changes

### PR DESCRIPTION
**Pull request by shontzu-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/core/src/App/Constants/platform-config.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ const platform_config = [
     },
     {
         icon: getPlatformSettings('bbot').icon,
-        title: () => getPlatformSettings('bbot').name,
+        title: () => getPlatformSettings('bbotDark').name,
```
Comment (thisyahlen-deriv) [inline packages/core/src/App/Constants/platform-config.js]: I dont think this is necessary. You can rename it back to bbot 👁️

File: packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx [hunk 2]
```diff
@@ -158,11 +158,15 @@ const TradingHubHeader = ({
...
-                ) : (
-                    <DerivBrandLogo className='trading-hub-header__logo' />
-                )}
+
+                <div className='platform-switcher'>
+                    {is_dark_mode ? (
+                        <DerivBrandLogo className='trading-hub-header__logo' />
+                    ) : (
+                        <DerivBrandLogoDark className='trading-hub-header__logo' />
+                    )}
```
Comment (thisyahlen-deriv) [inline packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx]: `{is_dark_mode ? <DerivBrandLogo className='trading-hub-header__logo' /> : <DerivBrandLogoDark className='trading-hub-header__logo' />}
`

**Review by farzin-deriv — COMMENTED** [review]
@shontzu-deriv I would suggest the same thing as I did [here](https://github.com/binary-com/deriv-com/pull/4135#discussion_r1096510077) for the images.

File: packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx [hunk 3]
```diff
@@ -4,27 +4,20 @@ import PropTypes from 'prop-types';
...
-            data_testid='dt_platform_box_icon'
-            className='platform-dropdown__list-platform-icon'
-            icon={icon}
-            size={32}
-        />
 
-        <div className='platform-dropdown__list-platform-details'>
-            <p className='platform-dropdown__list-platform-title'>{title()}</p>
+        <div className='platform-switcher__dropdown'>
+            <Icon icon={is_dark_mode ? icon : `${icon}Dark`} height={42} width={150} />
```
Comment (hirad-deriv) [inline packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx:14]: if it is dark mode you are using normal icon, and in the else condition you use dark mode in your condition. You need to change this.

File: packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx [hunk 4]
```diff
@@ -4,27 +4,20 @@ import PropTypes from 'prop-types';
 import { Div100vhContainer, Icon, useOnClickOutside } from '@deriv/components';
 import { routes, isDesktop, isMobile, getActivePlatform, getPlatformSettings } from '@deriv/shared';
 import { BinaryLink } from 'App/Components/Routes';
-
 import 'Sass/app/_common/components/platform-dropdown.scss';
 
-const PlatformBox = ({ platform: { icon, title, description } }) => (
+const PlatformBox = ({ platform: { icon, description }, is_dark_mode }) => (
```
Comment (hirad-deriv) [inline packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx:9]: add your prop to proptypes.

File: packages/core/src/App/Components/Layout/Header/platform-switcher.jsx [hunk 5]
```diff
@@ -60,12 +62,14 @@ const PlatformSwitcher = ({
             >
                 <Icon
                     className='platform-switcher__icon'
-                    icon={getPlatformInformation(app_routing_history).icon}
-                    size={32}
+                    icon={
+                        is_dark_mode
```
Comment (hirad-deriv) [inline packages/core/src/App/Components/Layout/Header/platform-switcher.jsx:66]: Same issue here.

File: packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx (hunk 3 above)
Comment (hirad-deriv) [reply packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx:14]: I also see some changes to some of our icons (Deriv Go and ...) but they are still the same. Can you explain what changes have you done ?

Comment (shontzu-deriv) [reply packages/core/src/App/Components/Layout/Header/platform-dropdown.jsx:14]: @hirad-rewok I copied all the icons under the `brands` directory from an old PR into this PR during the cleanup

**Review by hirad-deriv — APPROVED** [review]

**Review by matin-deriv — CHANGES_REQUESTED** [review]
You do not need 2 different svgs for light/dark theme.
You can update the color using `var` keyword in the svg file to automatically support both colors.

**Review by matin-deriv — COMMENTED** [review]
Logo texts are missing.

**Comment by shontzu-deriv** [comment]
> Logo texts are missing.

the texts are back

**Review by yashim-deriv — APPROVED** [review]

**Review by matin-deriv — APPROVED** [review]
LGTM, you can merge first but need to remove dark mode icons and replace the fill color on normal svgs to a var variable. You can find examples in the code such as `<path fill="var(--fill-color1)" />`

**Pull request merged**